.eslintrc

```diff
@@ -238,9 +239,20 @@ rules:
   # Disallows use of leading or trailing decimal points in numeric literals.
   no-floating-decimal: 2
 
+  # Disallow the type conversions with shorter notations.
+  no-implicit-coercion:
+    - 1
+    -
+      boolean: true
```

Comment:
I'm okay with updating this to `false` to exclude boolean in the list. The other bit this catches, `var b = ~foo.indexOf(".");`, I've never actually seen in the wild.

@KimberlyMunoz do you have a preference? Warn on `!!var` to update to `Boolean( var )` or not?